docs/magento2/commerce.mdx

```diff
@@ -185,3 +212,180 @@ filters on ` option_id` instead of `value_id`.
+based on a static label, for example
+
+```jsx title="src/web/theme/modules/User/Order/OrderReturns/ReturnedItemAttributeTable.js"
+import React from "react";
+import { FormattedMessage } from "react-intl";
+import Wysiwyg from "theme/modules/WysiwygV2";
+
+/**
+ * @typedef Attribute
+ * @property {string} label
+ * @property {TODO_TypeDef} value
+ */
+
+/**
+ * @type {React.FC<{ attributes:Attribute[] }>} props
+ */
+const ReturnedItemAttributeTable = (props) => {
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>
+            <FormattedMessage
+              id="modules.User.Order.OrderReturns.ReturnedItemAttributeTable.question"
+              defaultMessage="Question"
+            />
+          </th>
+          <th>
+            <FormattedMessage
+              id="modules.User.Order.OrderReturns.ReturnedItemAttributeTable.answer"
+              defaultMessage="Answer"
+            />
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        {props.attributes.map((attribute, i) => (
+          <tr key={attribute.label + `-${i}`}>
+            <td>{attribute.label}</td>
+            <td>
+              // remove-next-line
+              <Wysiwyg content={attribute.value} />
+              // add-start
+              {attribute.label === "Yes/No" ? (
+                attribute.value === "1" ? (
+                  "Yes"
+                ) : (
+                  "No"
+                )
+              ) : (
+                <Wysiwyg content={attribute.value} />
+              )}
+              // add-end
+            </td>
+          </tr>
+        ))}
+      </tbody>
+    </table>
+  );
+};
+
+export default ReturnedItemAttributeTable;
+```
+
+```mdx-code-block
+  </div>
+</details>
+```
+
+<!--------------------DETAILS-------------------->
+
+```mdx-code-block
+<details>
+  <summary className="flex justify-between"><span>The <code>File and Image</code> fields are not supported</span><SinceVersion inline tag="2.22.1" /></summary>
+  <div>
+```
+
+Magento is not yet ready to handle such files in its current state.
+
+```mdx-code-block
+  </div>
+</details>
+```
+
+<!--------------------DETAILS-------------------->
+
+```mdx-code-block
+<details>
+  <summary className="flex justify-between"><span>The <code>Text Area</code> field filter and min/max lengths</span><SinceVersion inline tag="2.22.1" /></summary>
+  <div>
+```
+
+1. When submitting a TextArea with html, the Magento API strips any html tags
+   regardless of the filter option.
+
+2. When setting a min/max length and saving the Text Area field, the values are
+   not saved.
+
+```mdx-code-block
+  </div>
+</details>
+```
+
+#### Returns Attributes
+
+You can learn more about configuring Returns Attributes on the
+[Magento Commerce documentation](https://experienceleague.adobe.com/docs/commerce-admin/stores-sales/order-management/returns/attributes-returns.html)
+
+##### Supported Input Types
+
+<SinceVersion tag="2.22.1" />
```

Comment:
that should be specified for each type because the Dropdown was supported before
that version.

Also, unless I'm mistaken this won't be backported to 2.22 so the version should
be 2.23